std_msgs/Header header
float32 delta_t
GtsamValue[] values
